ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -165,7 +166,7 @@ data Mempool m blk = Mempool {
                  -> m (MempoolAddTxResult blk)
 
       -- | Manually remove the given transactions from the mempool.
-    , removeTxs      :: [GenTxId blk] -> m ()
+    , removeTxs      :: NE.NonEmpty (GenTxId blk) -> m ()
```

Comment:
Could a `| null txs = pure ()` branch in the implementation have supplanted the need for this commit's relatively large diff footprint?

Reply:
Done